build.gradle

```diff
@@ -372,15 +371,13 @@ envs.each { env ->
 /* Cleanup non buildable modules */
 buildDeps.removeAll([
   tasks.findByName("studio-ui_build"),
-  tasks.findByName("plugin-maker_build"),
-  tasks.findByName("video-center-blueprint_build")
+  tasks.findByName("plugin-maker_build")
 ] as Task[])
 
 cleanDeps.removeAll([
   tasks.findByName("studio-ui_clean"),
   tasks.findByName("plugin-maker_clean"),
-  tasks.findByName("test-suite_build"),
-  tasks.findByName("video-center-blueprint_clean")
+  tasks.findByName("test-suite_build")
```

Comment:
Why keep this? I thought we took `test-suite` out already, no?

Reply:
not completely, @yacdaniel is it ok to remove the repo now?

Reply:
yes we can remove it @jdrossl